utils/stripe_utils.py

```diff
@@ -55,6 +59,10 @@ class StripeUtils(object):
     CURRENCY = 'chf'
     INTERVAL = 'month'
```

Comment:
Two things here that I think may not have been clear.

1. Interval here is a Stripe metadata for a plan. 
```
One of 'day', 'week', 'month' or 'year'. The frequency with which a subscription should be billed. 
```
https://stripe.com/docs/api#plan_object-interval
We said that for now, we only support monthly payments.

2. Stripe has cool proration policy by default [i].
Stripe attempts prorated payments (subscriptions that have changed), at the start of the billing period. So far for us, I see that the change in the subscription is only possible when a user cancels it. For this case, we need to generate a Stripe invoice [ii] based on the number of days remaining in the billing period for which the user made the prepayment and make the refund. This is not yet implemented.
 [i] : https://stripe.com/docs/subscriptions/upgrading-downgrading#understanding-proration
 [ii] : https://stripe.com/docs/subscriptions/invoices